### DISCUSSION TITLE
The "pip install discord.py" command was not working

### DISCUSSION THREAD

**Discussion by bommerboltgaming** [Rapptz/discord.py#5952] [Q&A]
Please help I want to make a bot but I cannot access. I do not know how to download or install please help.

**Comment by bommerboltgaming** [comment]
here is a screenshot
<img width="158" alt="Screenshot 2020-10-19 150910" src="https://user-images.githubusercontent.com/72145372/96500753-525b5180-121d-11eb-9559-f3592c38bf8a.png">

**Comment by Sheppsu** [comment]
Are you in a Python Shell? pip is a command used in the command prompt

**Reply by bommerboltgaming** [reply]
where is the command prompt

**Reply by nxtlo** [reply]
@bommerboltgaming you press your windows button on your keyboard then type `cmd` then run `command prompt` then type `pip install discord.py`

**Comment by GrzegorzKrug** [comment]
Did you named your file `discord.py` ? 
for me it looks like you are calling pip inside python terminal, but you want to do it in system terminal
